ui/components/app/token-cell/token-cell.test.js

```diff
@@ -7,6 +7,13 @@ import { renderWithProvider } from '../../../../test/lib/render-helpers';
 import { useTokenFiatAmount } from '../../../hooks/useTokenFiatAmount';
```

Comment:
The changes in this file seem to go beyond the scope of the PR

Reply:
It was breaking because we introduced the hexToDecimal conversion in a related component.